modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -29,6 +29,9 @@
 #elif defined(_MSC_VER)
 #pragma warning(push)
 #pragma warning(disable : 4100)
+# if _MSC_VER < 1900
+#  pragma warning(disable:4268) // Disable warnings of ngraph. OpenVINO recommends use MSVS 2019.
```

Comment:
`1900` is MSVS 2015

This condition doesn't work at all.

Reply:
"1900 is MSVS 2015" - I know. Local build was successful with this condition. But OV package was fresher.
Sorry for experiment.